network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html

```diff
@@ -105,34 +105,45 @@ <h2>{{ page.partner_heading }}</h2>
       </div>
     </div>
   </div>
-<div class="container">
 
-<div class="row">
-  <div class="col-12">
-    <h2>{{ page.take_action_title }}</h2>
-  </div>
-  <div class="col-8">
+<div class="bg-light take-action-section">
+  <div class="container">
     <div class="row">
-      {% for item in page.take_action_cards.all %}
-        <a href="{{ item.link }}" class="col-6">
-          {% image item.image fill-400x200 as img %}
-          <img src="{{ img.url }}" alt="" />
-          <div>
-            {{ item.text }}
+      <div class="col-12">
+        <h4 class="take-action-title">{{ page.take_action_title }}</h4>
+      </div>
+      <div class="col-md-8">
+        <div class="row">
+          {% for item in page.take_action_cards.all %}
+            <div class="col-lg-6 d-flex align-items-stretch">
+              <div class="card mb-4">
+                {% image item.image fill-350x130 as img %}
+                  <img src="{{ img.url }}" class="card-img-top" alt="{{ img.alt }}">
+                  <div class="card-body">
+                    <a href="{{ item.internal_link.url }}" class="stretched-link take-action-link">
+                      <p class="card-text h5-heading">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
```

Comment:
This hardcoded placeholder text should be updated with content from the CMS.

Reply:
LOL This is why we do peer review, thank you for catching this!